spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/ConfigCheck.scala

```diff
@@ -56,9 +64,10 @@ object ConfigCheck {
    *
    * Fuzziness is determined by MatchThreshold
    */
-  def getSuggestedVars(unknownProp: String): Option[Seq[String]] = {
+  def getSuggestedVars(unknownProp: String, extraProps: Set[String] = Set.empty): Option[Seq[String]] = {
```

Comment:
Oh yes, it caught my eye during review, too, however it turned out there was an *intended* semantic difference. Now when I look at it for the second time, I think it *can* be simplified to Seq[String], it just needs some minor modification of the call-site.